### PR TITLE
Add dbus-1-glib to SLE12 to allow onboarding of JeOS SLE12SP5 clients

### PR DESCRIPTION
**Pull request by juliogonzalez**
# WARNING
To be tested by @rjmateus. @rjmateus if more packages are needed, add it to the list but do not merge the PR: I will review it on Monday just in case :-)

## What does this PR change?

Add dbus-1-glib to SLE12 to allow onboarding of JeOS SLE12SP5 clients

SLE12SP5 does not ship this package preinstalled on JeOS anymore.

## GUI diff

No difference.

- [x] **DONE**

## Documentation
- No documentation needed: Not needed. Fix.

- [x] **DONE**

## Test coverage
- No tests: Not covered until we have sumaform & testsuite covering all possible clients :-)

- [x] **DONE**

## Links

https://github.com/SUSE/spacewalk/issues/11177

- [x] **DONE**

## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_lint_checkstyle"		 
- [ ] Re-run test "java_pgsql_tests"		 
- [ ] Re-run test "ruby_rubocop"
- [ ] Re-run test "schema_migration_test_oracle"
- [ ] Re-run test "schema_migration_test_pgsql"		 
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"		 
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

File: susemanager/src/mgr_bootstrap_data.py [hunk 1]
```diff
@@ -202,6 +202,10 @@
     "python-setuptools",
 ]
 
+ENHANCE12SP5 = [
+    "dbus-1-glib",
```
Comment (mcalmer) [inline susemanager/src/mgr_bootstrap_data.py:206]: Is this package new on SP5? I think not.
Would it hurt when we copy it always?

Comment (mcalmer) [reply susemanager/src/mgr_bootstrap_data.py:206]: @rjmateus when you install this package on JeOS, does it install other packages as well?
We need to list all the dependencies of this one.

Comment (juliogonzalez) [reply susemanager/src/mgr_bootstrap_data.py:206]: > Is this package new on SP5? I think not.

No, not new. But on SP5 not JeOS and on previous SLE12 versions (JeOS and not JeOS) it is installed by default as part of the OS. It's a requirement of `dbus-1-python` which is listed at `PKGLIST12`.

So this is only for JeOS SLE12SP5.

> Would it hurt when we copy it always?

Only in terms of diskspace, FMPOV. If the package is installed on the client, our version will not get installed (for other SPs and 12SP5 non-JeOS).

So if you are happy with having it at all bootstrap repos duplicated, I can add it to `PKGLIST12`, of course :-)

Comment (juliogonzalez) [reply susemanager/src/mgr_bootstrap_data.py]: As discussed on rocket.chat, let's keep it simple.

**Comment by juliogonzalez** [comment]
@mcalmer yeah, it's no the description so Ricardo can pay attention to that.

**Comment by rjmateus** [comment]
@mcalmer @juliogonzalez I have tested the bootstrap script changes and add several transitive dependencies needed by "dbus-1-glib".
They are several but all these packages are installed in other sles12sp5 images, so I think it is ok to add it to PKGLIST12.
Another solution would be create a specific ENHANCE12SP5 just with these packages.

**Comment by mcalmer** [comment]
> @mcalmer @juliogonzalez I have tested the bootstrap script changes and add several transitive dependencies needed by "dbus-1-glib".

Shit, thats a lot new packages. Let's discuss on Tuesday what we do.

**Comment by rjmateus** [comment]
@juliogonzalez I just test it on aws and after added this changes the client was successfully registered.
I test it by including the changes in this PR on file `/usr/share/susemanager/mgr_bootstrap_data.py` 

From my side, is good to go.

**Comment by juliogonzalez** [comment]
Fine. Merging and tagging :-)

**Pull request merged**